packages/marshal/marshal.js

```diff
@@ -712,7 +745,8 @@ function Remotable(iface = 'Remotable', props = {}, remotable = {}) {
   };
 
   // Fail fast: check a fresh remotable to see if our rules fit.
-  mutateHardenAndCheck(Object.create(oldRemotableProto));
+  const throwawayRemotable = Object.create(oldRemotableProto);
+  mutateHardenAndCheck(throwawayRemotable);
```

Comment:
I see. This code is correct as is. One subtlety I was missing is why you need to add the properties from `props` just to check. Will this change in the later stages? (I think so).